doc/source/tutorial/stats/sampling_pinv.rst

```diff
@@ -173,22 +174,22 @@ PDF evaluations increase during setup for small values of ``u_resolution``.
     >>> rng = NumericalInversePolynomial(dist, u_resolution=1e-8,
     ...                                  random_state=urng)
     >>> dist.callbacks
-    4095
+    4095        # may vary
     >>> dist.callbacks = 0  # reset the number of callbacks
     >>> # u_resolution = 10^-10 (default)
     >>> # => more PDF evaluations required
     >>> # => slow setup
     >>> rng = NumericalInversePolynomial(dist, u_resolution=1e-10,
     ...                                  random_state=urng)
     >>> dist.callbacks
-    11454
+    11454       # may vary
```

Comment:
@tirthasheshpatel git blame point to you in these sampling_*rst files. Could you take a look over my tweaks to see if they mask real issues?